core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -327,7 +325,7 @@ object Boot extends IOApp with LazyLogging {
             executionServiceServers.map(c => c.key -> c.dao).toMap,
             groupsToCheck = Seq(gcsDAO.adminGroupName, gcsDAO.curatorGroupName),
             topicsToCheck = Seq(gcsConfig.getString("notifications.topicName")),
-            bucketsToCheck = Seq(gcsDAO.tokenBucketName)
+            bucketsToCheck = Seq.empty //todo: find a bucket that we can use here
```

Comment:
highlighting this todo with a github comment so it doesn't slip through

Reply:
note to self: maybe use this one https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/rawls/rawls.conf.ctmpl#L384